Blur canvas

majil334 opened the pull request:
Har lagt till funktionaliteten att canvasen blurras och en knapp man kan stänga av och sätta på blurren med.

Commit:
Started to implement data layer buttons

Commit:
Har testat lite

Commit:
Har testat lite

Commit:
Sparar bara för att jobba med en annan branch

Commit:
Sparar bara för att jobba med en annan branch

Commit:
continued with buttons

Commit:
continued with buttons

Commit:
chose how the data-layer state should be stored

Commit:
commit för att ladda ner master

Commit:
commit för att ladda ner master

Commit:
commit för att kolla på PR

Commit:
commit för att kolla på PR

Commit:
ground structure for buttons. need to test data-layer-set so it works correctly.

Commit:
Klar men master har blivit uppdaterad, merge innan PR

Commit:
Klar men master har blivit uppdaterad, merge innan PR

Commit:
corrected formatting

Commit:
corrected formatting

Commit:
Data-layer-buttons work correctly. Data-layers are stored, in an atom called data-layer-atom, as a set. When a button is pressed that corresponding data layer is added to the set. If the button is pressed again, the data layer is removed. If you open the console in your browser the data layer set is printed every time a button is pressed. In that way you can check that the buttons work correctly.

Commit:
Fixed syntax with cljfmt

Rovanion committed:
Changed from ssh url to https url for heroku nginx buildpack.

majil334 committed:
Merge pull request #13 from WeatherMagic/data-layer-buttons

Data layer buttons, works good for me!

majil334 committed:
Merge pull request #14 from WeatherMagic/heroku-url-fix

Changed from ssh url to https url for heroku nginx buildpack.

Rovanion commented on resources/public/css/style.css:
```diff
@@ -16,6 +16,19 @@ input[type=range] {
+  height: 100vh;
+  width: 100vw;
+  top: 0;
+  background-color: grey;
```
Här ska det vara svart istället för grått. Grått innehåller vitt så det ser lite konstigt ut, vi vill bara mörka ut bakgrunden och justera hur mycket med opacity.

Rovanion commented:
Åter igen har du fått med en hel bunt .DS_Store. Använd inte `git add *` på Mac för då får du med dem.

Jag borde lägga till det i .gitignore nu när jag tänker på det, men det förändrar inte problemet i den här existerande branchen.

Rovanion commented:
Vidare är en fil earchM.jpg inkluderad som inte borde vara med.

Rovanion commented on src/weather_magic/core.cljs:
```diff
@@ -85,7 +85,7 @@
   (-> M44
       (g/rotate-x (m/radians 24.5))
-      (g/rotate-y (/ t 10))))
+      (g/rotate-y (/ t t))))
```
Rotera i y-led med t/t?

Rovanion commented on src/weather_magic/core.cljs.orig:
```diff
@@ -0,0 +1,167 @@
+(ns weather-magic.core
```
Hela den här .orig-filen ska bort.

Commit:
removed DS files

Commit:
removed DS files

Rovanion commented on src/weather_magic/ui.cljs:
```diff
@@ -23,12 +23,30 @@
+
+(def blur-visible (atom :visible))
+
+(defn blur-obj []
```
blur-obj är inte uppenbart vad det är för något. Kanske vi kan döpa det det map-ui-blur? För det är väl precis map-ui vi ska täcka och inte de andra UI-delarna?

Men det kan finnas bättre namn att döpa den till.

Commit:
removed orig file

Commit:
removed orig file

Commit:
gjorde önskade ändringar

Commit:
gjorde önskade ändringar

Commit:
ändrade min branch för master hade uppdaterats
Merge branch 'blurr-canvas' of https://github.com/WeatherMagic/weather-front into blurr-canvas

Commit:
fixed formatting
